code/modules/detectivework/tools/sample_kits/_sample.dm

```diff
@@ -0,0 +1,56 @@
+
+/obj/item/forensics/sample
+	name = "forensic sample"
+	icon = 'icons/obj/forensics.dmi'
+	var/object
+	var/list/possible_evidence_types = list(/datum/forensics/fibers)
+	var/list/evidence
```

Comment:
Not entirely sure, but you may need to selectively qdel elements in here on Destroy if they are instances.

Reply:
qdeling all that crap now